playbook/lib/playbook/kit_base.rb

```diff
@@ -122,7 +122,9 @@ def combined_html_options
       html_options.each do |key, value|
         if key == :style && value.is_a?(Hash)
           # Convert style hash to CSS string
+          # rubocop:disable Performance/StringReplacement
```

Comment:
Got lint errors on 5 month old code? not sure why but since this is just a POC and not shipping anything i just disabled it on the PR